internal/database/sqlcommon/namespace_sql.go

```diff
@@ -161,6 +160,30 @@ func (s *SQLCommon) GetNamespace(ctx context.Context, name string) (message *fft
 	return namespace, nil
 }
 
+func (s *SQLCommon) GetNamespaceByID(ctx context.Context, id *fftypes.UUID) (ns *fftypes.Namespace, err error) {
```

Comment:
Could we follow the pattern here and remove a lot of the code/test duplication?

https://github.com/hyperledger/firefly/blob/1c0672cb052128dbb0c18d14003b3ee0e81b9891/internal/database/sqlcommon/datatype_sql.go#L160-L166